<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="VentanaControlAsistencia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control de asistencia</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="botonSalir">
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="botonRegistrarLlegada">
     <property name="text">
      <string>Registrar llegada</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="botonRegistrarSalida">
     <property name="text">
      <string>Registrar salida</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
